services/src/main/java/io/grpc/services/BinlogHelper.java

```diff
@@ -83,6 +83,13 @@
   private static final Logger logger = Logger.getLogger(BinlogHelper.class.getName());
   private static final boolean SERVER = true;
   private static final boolean CLIENT = false;
+  // This is a special key that is set from above the application API,
+  // despite having a 'grpc-' prefix. Normally 'grpc-' metadata keys are set from within
+  // gRPC.
+  static final Metadata.Key<byte[]> STATUS_DETAILS_KEY =
```

Comment:
Shouldn't this be snuck from `StatusProto`?

Reply:
The StatusProto version of this turns it into an actual proto object, but we only care about the bytes. Also, that Key is private which is inconvenient to deal with.

Reply:
Can you comment saying the name must be identical to the actual key location?

Reply:
You mean the fact that "grpc-status-details-bin" is not some arbitrary name, but is instead a special well known key?

Reply:
yes.  It needs to be clear that grpc- is a special